From bb1d5595b63c7cdbae53763d229fe53b49fc916f Mon Sep 17 00:00:00 2001
From: Brad Love <brad@nextdimension.cc>
Date: Tue, 14 May 2019 15:57:06 -0500
Subject: [PATCH 2/3] Turn off tagged build, modinfo must match

---
 scripts/setlocalversion | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/setlocalversion b/scripts/setlocalversion
index 63d91e2..a6561cc 100755
--- a/scripts/setlocalversion
+++ b/scripts/setlocalversion
@@ -14,7 +14,7 @@ usage() {
 	exit 1
 }
 
-scm_only=false
+scm_only=true
 srctree=.
 if test "$1" = "--save-scmversion"; then
 	scm_only=true
-- 
2.7.4

