From b2d3a88f01ccb3ca329df15446403a75ac1e582c Mon Sep 17 00:00:00 2001
From: Brad Love <brad@nextdimension.cc>
Date: Fri, 10 May 2019 14:58:50 -0500
Subject: [PATCH 3/4] Turn off tagged build, modinfo must match

---
 scripts/setlocalversion | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/setlocalversion b/scripts/setlocalversion
index 84b88f1..079de43 100755
--- a/scripts/setlocalversion
+++ b/scripts/setlocalversion
@@ -14,7 +14,7 @@ usage() {
 	exit 1
 }
 
-scm_only=false
+scm_only=true
 srctree=.
 if test "$1" = "--save-scmversion"; then
 	scm_only=true
-- 
2.7.4

